data-raw/internal_data.R

```diff
@@ -5,9 +5,10 @@
 col_palette <- c("#1a9641", "#a6d96a", "#fdae61", "#d7191c", "grey50")
 names(col_palette) <- c("within limits", 
                         "out of limits, clinically insignificant", 
-                        "out of limits, significance unknown", 
+                        "out of limits, significance pending", 
                         "out of limits, clinically significant", 
                         "limits unknown")
+usethis::use_data(col_palette, overwrite = TRUE)
```

Comment:
Note: this line was needed to update `data/col_palette.rda`. I'm not sure if it's used anymore, but I wanted to make sure it got updated too. `col_palette` is still present in `sysdata.rda` too.